internal/tiltfile/live_update.go

```diff
@@ -203,7 +202,7 @@ func (s *tiltfileState) liveUpdateStepToModel(t *starlark.Thread, l liveUpdateSt
 	case liveUpdateFallBackOnStep:
 		return model.LiveUpdateFallBackOnStep{Files: x.files}, nil
 	case liveUpdateSyncStep:
-		if !filepath.IsAbs(x.remotePath) {
+		if !unixIsAbs(x.remotePath) {
```

Comment:
I think we want to use [`path.isAbs`](https://golang.org/src/path/path.go?s=5145:5173#L191) (note `path` vs `filepath`):
```suggestion
		if !path.IsAbs(x.remotePath) {
```

This should ensure that a Unix definition of absolute path is used.

Reply:
+1